src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/integration/endtoend/AuditTest.kt

```diff
@@ -68,13 +61,54 @@ class AuditTest {
       },
       isNull()
     )
-    verify(auditRepository, times(1)).save(any(HMPPSAuditListener.AuditEvent::class.java))
+    verify(auditRepository, times(1)).save(any(AuditEvent::class.java))
   }
 
-  fun getNumberOfMessagesCurrentlyOnQueue(): Int? {
-    val queueAttributes = awsSqsClient.getQueueAttributes(queueName.queueUrl(), listOf("ApproximateNumberOfMessages"))
-    return queueAttributes.attributes["ApproximateNumberOfMessages"]?.toInt()
-  }
+  @Suppress("ClassName")
+  @Nested
+  open inner class insertAuditEntries {
+    @Test
+    fun `save basic audit entry`() {
+      webTestClient.post()
+        .uri("/audit")
+        .headers(setAuthorisation(roles = listOf("ROLE_AUDIT")))
+        .contentType(MediaType.APPLICATION_JSON)
+        .body(BodyInserters.fromValue(AuditEvent(what = "basicAuditEvent")))
+        .exchange()
+        .expectStatus().isCreated
+
+      AuditService.log.info(auditRepository.hashCode().toString())
+
+      await untilCallTo { mockingDetails(auditRepository).invocations.size } matches { it == 1 }
+
+      verify(telemetryClient).trackEvent(eq("hmpps-audit"), com.nhaarman.mockitokotlin2.any(), isNull())
```

Comment:
It looks like you're using the original mockito `any` in this class too - I think you only need the nhaarman version?